src/ophys_etl/workflows/on_prem/dags/ophys_processing_trigger.py

```diff
@@ -0,0 +1,119 @@
+import base64
+import datetime
+import logging
+from typing import List, Optional
+
+import requests
+from airflow.decorators import task
+from airflow.models.dag import dag
+from airflow.operators.python import get_current_context
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+from ophys_etl.workflows.app_config.app_config import app_config
+
+from ophys_etl.workflows.utils.lims_utils import LIMSDB
+
+# List of queues in LIMS that should trigger ophys processing
+LIMS_OPHYS_PROCESSING_TRIGGER_QUEUES = (
+    'MESOSCOPE_FILE_SPLITTING_QUEUE',
```

Comment:
I think the only real one we will have to worry about is `MESOSCOPE_FILE_SPLITTING_QUEUE`. The others I believe are for retired rigs, but we should confirm that with Marina or the like.

Reply:
I parameterized it so that we can pass it at runtime